src/backend/core/tasks/metrics.py

```
@@ -125,6 +125,35 @@ def scrape_service_metrics(service_id: int):
         return {"status": "error", "message": str(e)}
 
 
+def scrape_service_usage_metrics(service: Service, filters: Dict[str, Any] = {}) -> List[Dict[str, Any]]:
+    """
+    Scrape usage metrics for a specific service (Celery task).
+    """
+    metrics_data = fetch_usage_metrics_from_service(service, filters)
+    if metrics_data:
+        store_service_usage_metrics(service, metrics_data)
+
+def fetch_usage_metrics_from_service(service: Service, filters: Dict[str, Any] = {}) -> List[Dict[str, Any]]:
```

Ruff (0.14.3)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)